otpprofiler.py

```diff
@@ -148,18 +149,59 @@ def get_params(fast, count, filename="requests.json", requests_json=None, modes=
         req['tid'] = target['id']
         req['fromPlace'] = "%s,%s" % (origin['lat'], origin['lon'])
         req['toPlace'] = "%s,%s" % (target['lat'], target['lon'])
-        req['walkSpeed'] = 1.222
 
+        from_location = {"coordinate": {
+            "latitude": float(req['fromPlace'].split(',')[0]),
+            "longitude": float(req['fromPlace'].split(',')[1])
+            }}
+
+        to_location = {"coordinate": {
+            "latitude": float(req['toPlace'].split(',')[0]),
+            "longitude": float(req['toPlace'].split(',')[1])
+        }}
+        d = list(map(int, Date.split('-')))
+        t = list(map(int, Time.split(':')))
+
+        dt = datetime(d[0], d[1], d[2], t[0], t[1], tzinfo=ZoneInfo("Europe/Helsinki"))
+        offset_datetime = dt.strftime("%Y-%m-%dT%H:%M:%S%z")
+        offset_datetime = offset_datetime[:-2] + ":" + offset_datetime[-2:]
+        latest_arrival = req.get('arriveBy', False)
+        datetime_key = 'latestArrival' if latest_arrival else 'earliestDeparture'
+        modes = req['mode'].split(',')
+
+        if req['mode'] in ('WALK','BICYCLE'):
+            dist = vincenty_inverse((origin['lat'], origin['lon']),(target['lat'], target['lon']))
+            if dist > (0.9/1000)*req['maxWalkDistance']:
+                modes.append('TRANSIT')
+
+        direct_only = 'TRANSIT' not in modes
+
+        direct = modes if direct_only else None
+        if 'TRANSIT' in modes:
+            modes.remove('TRANSIT')
+
+        transit = {
+            'access': modes if  not direct_only else 'WALK',
+            'egress': modes if not direct_only else 'WALK',
+            'transfer': modes if not direct_only else 'WALK',
+        }
+
+        plan_modes_input = {
+            'directOnly': direct_only,
+            'direct': direct,
+            'transit': transit,
+        }
```

Comment:
Btw what would happen if we added `transitOnly: !direct_only` here? It could make sense since we want to test that the transit network works.